docs/source/cheat-sheet/index.rst

```diff
@@ -42,3 +42,24 @@ Here you can see a couple of usefull ROS commands to help you out!
 * :bash:`$ rosrun rqt_top rqt_top`
```

Comment:
I don't think you meant to commit that.

Reply:
The changes made here are already present in the main index.rst and should not be part of the ros cheat-sheet doc.

Reply:
It was already there Eric and its been there for a while. If you want to remove it, I feel like that should be the subject of another PR.